### ISSUE TITLE
Conflicting names

### ISSUE THREAD

**Issue by mikejohnson51** [dblodgett-usgs/hyRefactor#40]
Both `hyRefactor` and `hyAggregate` have an `aggregate_network` function. I think there are two solutions to provide more clarity. 

Since `hyRefactor` is a `hyAggregate` dependency, the point based aggregation workflow could be moved to `hyAggregate` and the aggregation driven by the input parameters. This would result in a signature like:

```
aggregate_network = function(gf = NULL,
                             flowpaths  = NULL,
                             catchments = NULL,
                             ideal_size_sqkm = 10,
                             min_length_km = 1,
                             min_area_sqkm  = 3,
                             outfile = NULL,
                             verbose = TRUE,
                             overwrite = FALSE,
                             nexus_topology = TRUE,
                             routelink_path = NULL,
                             outlets,
                             only_larger = FALSE) {
 
... 
}
```
Or, the functions could be renamed to something like: `aggregate_network_to_distribution` and `aggregate_network_to_outlets` and either remain in the separate packages, or, consolidate into `hyAggregate`. 

Do you have any thoughts?

Mike

**Comment by dblodgett-usgs** [comment]
I'd be happy to have it move to hyAggregate -- honestly, I think the packages could combine long run.

**Comment by mikejohnson51** [comment]
I agree - next time we talk this would be any interesting point to discuss. The hyAggregate dependency list has bloated a bit to support nextgen specific tasks. It might be time to bring hyRelease back into the mix...

Either way, you want me to bring the hyRefactor function over or do you want to submit a PR? I have a _big_ push coming this week so there is no urgency.
